_layouts/galatour.html

```diff
@@ -0,0 +1,163 @@
+---
+layout: default
+---
+<div id="tour-banner">
+    <div class="container">
+        <div class="row">
+            <div class="col-12 mb-4 col-md-6 col-lg-6 align-self-center">
+                <h1>2023春节游园会</h1>
+                <div><br>
+                    <p>
+                        时间：2023年1月21日（周六）， 13:00 -16:00 <br>
+                        地点： San Jose Center for the Performing Arts <br>
+                    </p>
+                </div>
+            </div>
+        </div>
+    </div>
+</div>
+
+<!-- Tab links -->
+<div class="tab">
+    <button class="tab-links default-open" onclick="openTab(event)" data-tab-group="main" data-tab-id="aboutGalatour">关于春节游园会</button>
+    <button class="tab-links" onclick="openTab(event)" data-tab-group="main" data-tab-id="stageGalatour">游园舞台节目招募</button>
+    <button class="tab-links" onclick="openTab(event)" data-tab-group="main" data-tab-id="mapGalatour">游园地图</button>
+</div>
+
+<!-- Tab content -->
+<div id="aboutGalatour" class="tab-content">
+    <div class="container pt-6 pb-6 pt-md-8 pb-md-8">
+        <div class="row">
+            <h1 style="text-align:center">什么是春节游园会</h1>
+            <p> 
+                叮咣叮咣，新年来到！远在异国他乡的你，是否也怀念过年各处张灯结彩的喜庆呢？找不到组织的你，是否也想沾沾年味热闹一番呢？<br>
+                民俗活动，传统游戏，精彩互动，就在“载歌在谷”欢乐游园！湾区规模最大，最热闹，最具中国特色的春节晚会。适合全家参与，赶快带小朋友来体验我中华民族的春节气息吧！活动不仅免费，还有各种礼品和奖品送！<br>
+            </p>
+        </div>
+        <div class="row">
+            <h1 style="text-align:center">游园剪影</h2>
+        </div>
+        <div class="row pb-6">
+            <div class="col">
+                <img src=https://res.cloudinary.com/zaigezaigu/image/upload/v1670183640/zgzg-io-website/%E5%89%AA%E5%BD%B1_z4ywki.png class="rounded mx-auto d-block">
+            </div>
+        </div>
+        <div class="row">
+            {% if site.data.data.galatour_picture_tour %}
+                {% for programme in site.data.data.galatour_picture_tour %}
+                    {% include programme.html %}
+                {% endfor %}
+            {% endif %}
+        </div>
+        <div class="row">
+            <h1 style="text-align:center">游园志愿者团队</h1>
+            <p> 
+                春节游园舞台志愿者可以在这些领域发挥才能：<br>
+            </p>
+        </div>
+        <div class="row">
+            {% if site.data.data.galatour_picture_volunteer %}
+                {% for programme in site.data.data.galatour_picture_volunteer %}
+                    {% include programme.html %}
+                {% endfor %}
+            {% endif %}
+        </div>
+        <div class="row">
+            <h1 style="text-align:center">春节游园舞台志愿者报名</h1>
+            <p> 
+                春节游园舞台志愿者、节目报名如有任何问题可联系舞台负责人Jay。 Email: jayz@zgzg.io<br>
+                【报名时间】即日起至2022年11月1日 11:59pm PST<br>
```

Comment:
时间格式要统一。前面用的是24小时制，这里就最好也要用。

另外，直接说PT，可以避免需要考虑PDT、PST的区别。

```suggestion
                【报名时间】即日起至 2022 年 11 月 1 日 23:59 PT<br>
```